### ISSUE TITLE
Could you please include our paper in the "Knowledge from logits" section?

### ISSUE THREAD

**Issue by marcomistretta** [FLHonker/Awesome-Knowledge-Distillation#29]
Hi,

My name is Marco Mistretta, and I am a researcher at [MICC (Florence, Italy)](https://github.com/miccunifi).
Thank you for creating this repository! It has been an invaluable resource for me.

Could you please add our ECCV24 paper, [Improving Zero-shot Generalization of Learned Prompts via Unsupervised Knowledge Distillation](https://arxiv.org/abs/2407.03056)?
It's the first time that unsupervised knowledge distillation has been applied to prompt learning in the few-shot setting!

Thank you for your time and consideration!

TITLE: Improving Zero-shot Generalization of Learned Prompts via Unsupervised Knowledge Distillation
CONFERENCE: ECCV24
PAPER: https://arxiv.org/abs/2407.03056
CODE: https://github.com/miccunifi/KDPL
